.github/workflows/ci_cd.yml

```
@@ -53,7 +53,7 @@ jobs:
     name: Doc dev version deploy
     if: ${{ github.ref == 'refs/heads/main' }}
```

This will avoid overriding the development documentation with the latest stable release:

```suggestion
    if: github.event_name == 'push' && !contains(github.ref, 'refs/tags')
```